src/components/Semester/Semester.vue

```diff
@@ -61,7 +61,7 @@
           group="draggable-semester-courses"
           v-model="coursesForDraggable"
           item-key="uniqueID"
-          :style="{ height: courseContainerHeight + 'rem' }"
+          :componentData="{ style: { height: courseContainerHeight + 'rem' } }"
```

Comment:
Do we need to do this to any other in-line styles in the codebase?

Reply:
No I think it just applies to this library.